src/content/popup/KanjiReferencesTable.tsx

```diff
@@ -0,0 +1,102 @@
+import { KanjiResult } from "@birchill/jpdict-idb";
+
+import { PopupOptions } from "./popup";
+import { getSelectedReferenceLabels } from "../../common/refs";
+import { getReferenceValue } from "../reference-value";
+import { I18nProvider, useLocale } from "../../common/i18n";
+
+type Props = {
+  entry: KanjiResult,
+  options: PopupOptions,
```

Comment:
I think it would be best to only pass in the actual options needed by this component. That will make writing fixtures easier and make for a more sensible component API too.

Reply:
Yeah, that seems sensible. Looks like we're only use `options.kanjiReferences` in this component anyway.